src/components/Header/util.tsx

```diff
@@ -1,5 +1,4 @@
-export const getDateFromMillisecond = (milliseconds: number) => {
-  const date = new Date(milliseconds);
+export const getDateFromMillisecond = (date: Date) => {
```

Comment:
Can we fix the function name from `getDateFromMillisecond` to `getParsedDate` or something more generic?